src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -3680,6 +3796,33 @@ public static IndividualResource setupMissingItem(ItemsFixture itemsFixture) {
     return missingItem;
   }
 
+  private ItemResource createItemWithStatus(ItemsFixture itemsFixture, String itemStatus) {
+    ItemResource item = null;
+    switch (ItemStatus.from(itemStatus)) {
+      case IN_TRANSIT:
+        item = itemsFixture.basedUponSmallAngryPlanet(ItemBuilder::inTransit);
+        assertThat(item.getResponse().getJson().getJsonObject("status").getString("name"),
+          is(ItemStatus.IN_TRANSIT.getValue()));
+        break;
+      case ON_ORDER:
+        item = itemsFixture.basedUponSmallAngryPlanet(ItemBuilder::onOrder);
+        assertThat(item.getResponse().getJson().getJsonObject("status").getString("name"),
+          is(ItemStatus.ON_ORDER.getValue()));
+        break;
+      case IN_PROCESS:
+        item = itemsFixture.basedUponSmallAngryPlanet(ItemBuilder::inProcess);
+        assertThat(item.getResponse().getJson().getJsonObject("status").getString("name"),
+          is(ItemStatus.IN_PROCESS.getValue()));
+        break;
+      case MISSING:
+        item = itemsFixture.basedUponSmallAngryPlanet(ItemBuilder::missing);
+        assertThat(item.getResponse().getJson().getJsonObject("status").getString("name"),
+          is(ItemStatus.MISSING.getValue()));
+        break;
+    }
+    return item;
+  }
```

Comment:
```suggestion
  private ItemResource createItemWithStatus(String itemStatus) {
    return itemsFixture.basedUponSmallAngryPlanet(
      builder -> builder.withStatus(itemStatus.getValue()));
  }
```

For this, you need to make method `api.support.builders.ItemBuilder#withStatus` public. No need to pass item fixture, it is an instance field on this test class.
You only use this method once. Can we just inline it?

Reply:
It is inlined, thank you.